src/burnchains/bitcoin/bits.rs

```diff
@@ -520,10 +520,9 @@ impl BitcoinTxInput {
 }
 
 fn to_txid(txin: &BtcTxIn) -> (Txid, u32) {
-    (
-        Txid(txin.previous_output.txid.0.clone()),
-        txin.previous_output.vout,
-    )
+    let mut bits = txin.previous_output.txid.0.clone();
```

Comment:
Might make a comment here about how the `bitcoin-rs` library works in a different endian-ness than we're used to.

Reply:
Added in https://github.com/blockstack/stacks-blockchain/pull/2129/commits/b4c7b8878ad442a81a5fe47b6e94de287de49305